Cargo.toml

```diff
@@ -35,6 +35,7 @@ systemd = { version = "0.8", default-features = false, optional = true }
 dbus = "0.9.2"
 tabwriter = "1"
 fastrand = "1.4.1"
+crossbeam-channel = "0.5"
```

Comment:
Is there any way to narrow down the features?
https://github.com/crossbeam-rs/crossbeam/blob/02e74146ca0b871974fccda579911bf57a9c0ea9/Cargo.toml#L18-L41

Reply:
I think we already do. If I understand how this crate works, there is crossbeam with feature ("crossbeam-channel/std" for example), or use `crossbeam-channel` directly. crossbeam with feature "crossbeam-channel/std" seems to point to `crossbeam-channel` underneath.

For context, to be honest, I only used crossbeam because ` std::sync::mpsc` documentation [here](https://doc.rust-lang.org/std/sync/mpsc/struct.Receiver.html#method.recv_timeout) mentioned [this unfixed issue](https://github.com/rust-lang/rust/issues/39364#issuecomment-880136715) where it mentioned `crossbeam` being a potential replacement for the ` std::sync::mpsc`,